inference-engine/ie_bridges/c/include/c_api/ie_c_api.h

```diff
@@ -43,6 +44,10 @@
     #endif
 #endif
 
+#ifndef INFERENCE_ENGINE_CALLBACK
+#define INFERENCE_ENGINE_CALLBACK
```

Comment:
Looks good, but can we have a name like `INFERENCE_ENGINE_C_API_CALLBACK` to have different naming convention for C and C++ APIs?